packages/client/components/TeamInvitation.tsx

```diff
@@ -1,17 +1,31 @@
-import React from 'react'
-import {createFragmentContainer} from 'react-relay'
 import graphql from 'babel-plugin-relay/macro'
-import {TeamInvitation_verifiedInvitation} from '../__generated__/TeamInvitation_verifiedInvitation.graphql'
+import React from 'react'
+import {PreloadedQuery, usePreloadedQuery} from 'react-relay'
+import {TeamInvitationRootQuery} from '../__generated__/TeamInvitationRootQuery.graphql'
 import TeamInvitationDialog from './TeamInvitationDialog'
-import TeamInvitationWrapper from './TeamInvitationWrapper'
 import TeamInvitationMeetingAbstract from './TeamInvitationMeetingAbstract'
+import TeamInvitationWrapper from './TeamInvitationWrapper'
 
 interface Props {
-  verifiedInvitation: TeamInvitation_verifiedInvitation
+  queryRef: PreloadedQuery<TeamInvitationRootQuery>
 }
 
 function TeamInvitation(props: Props) {
-  const {verifiedInvitation} = props
+  const {queryRef} = props
+  const data = usePreloadedQuery<TeamInvitationRootQuery>(
+    graphql`
+      query TeamInvitationRootQuery($token: ID!) {
```

Comment:
-1 see above, please rename, same in the files following

Reply:
sure, thanks Georg @Dschoordsch